styles.css

```diff
@@ -0,0 +1,10 @@
+.button {

```

Comment:
this is not linked to the button and will need to be refactored to link to the input change request